virttest/utils_v2v.py

```diff
@@ -606,8 +606,23 @@ def get_driver_info(self, signed=True):
         cmd = "DRIVERQUERY"
         if signed:
             cmd += " /SI"
-        status, output = self.session.cmd_status_output(cmd)
-        return self.run_cmd(cmd)[1]
+        # Try 5 times to get driver info
+        output, count = '', 5
+        while count > 0:
+            logging.debug('%d times remaining for getting driver info' % count)
+            try:
+                output = self.session.cmd_output(cmd)
+            except Exception as detail:
+                logging.error(detail)
+                count -= 1
+                # Clean up output
+                self.session.cmd('cls')
+            else:
+                break
+        if not output:
+            logging.error('Fail to get driver info')
+        logging.debug("Command output:\n%s", output)
+        return output
 
```

Comment:
Thanks for the advice. : )  Sorry I haven't expressed clearly, this command will fail and raise error, after catching this error, output stream should be cleaned, otherwise it would mix with the next output of command, and utils_misc.wait_for may not actually meet the need.